website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -1,163 +1,145 @@
-The `Dropdown` component is composed of different child components, each with their own APIs:
+## Component API
 
-*   the dropdown component (parent to the child components)
-*   Toggle components to open/close the dropdown
-    
-    *   ToggleButton
-    *   ToggleIcon
-*   And finally, list item components, to build the dropdown's list items
-    
-    *   Description
-    *   Generic
-    *   Interactive
-    *   Separator
-    *   Title
+The `Hds::Dropdown` component is composed of different child components, each with their own APIs:
 
-_Notice: to make the invocation more intuitive for developers, all the sub-components are named yields, so the yielded name is a simplified version of the full component name (eg. `Hds::ListItem::Interactive` becomes just `Interactive`). See below how they are invoked as yielded components._
+- The Dropdown component
+- Toggle components to open/close the dropdown
+  - ToggleButton
```

Comment:
These sub items probably need another level of ident for the markdown to be rendered correctly